sklearn/tests/test_multiclass.py

```diff
@@ -605,3 +605,13 @@ def test_ecoc_gridsearch():
     cv.fit(iris.data, iris.target)
     best_C = cv.best_estimator_.estimators_[0].C
     assert_true(best_C in Cs)
+
+def test_pairwise_attribute():
```

Comment:
This test is good though I'd prefer if there was (additionally?) a test that checks that the behavior is actually the one we want. Maybe check that ``cross_val_score`` works correctly? Also, you test only tests one of the two properties.